cogs/misc.py

```diff
@@ -179,6 +181,23 @@ async def edit(text: str) -> None:
 
         await edit(f"{COMPLETED_EMOJI} Presented {len(tables)} table(s).")
 
+    @commands.Cog.listener()
+    async def on_message(self: Misc, message: discord.Message) -> None:
+        """
+        It adds the :swaghi: reaction to a message if it's in the #introductions channel.
+
+        :param message: The message that was sent.
+        :type message: discord.Message
+        """
+        if (
+            message.channel.id == INTRODUCTIONS_CHANNEL_ID
+            and SWAGHI_EMOJI_ID is not None
+        ):
+            swaghi_emoji_id_int = int(SWAGHI_EMOJI_ID)
+            swaghiEmoji = self.bot.get_emoji(swaghi_emoji_id_int)
```

Comment:
snake_case please :)
```suggestion
            swaghi_emoji_id = int(SWAGHI_EMOJI_ID)
            swaghi_emoji = self.bot.get_emoji(swaghi_emoji_id)
```
or with the new env name
```suggestion
            emoji_id = int(AUTO_REACT_WITH_EMOJI_ID)
            emoji = self.bot.get_emoji(emoji_id)
```